build/package/windows/nri-386-installer/Product.wxs

```diff
@@ -49,51 +47,34 @@
         </DirectoryRef>
         <DirectoryRef Id="CustomPluginsFolder">
-            <Component Id="CMP_V1_CUSTOM_PLUGINS" Guid="e482e265-efdc-40be-bad3-6a091f412090" KeyPath="yes">
+            <Component Id="CMP_V1_CUSTOM_PLUGINS" Guid="0ee4aaf4-9923-499c-a1d0-ff55b205a76c" KeyPath="yes">
                 <CreateFolder/>
             </Component>
         </DirectoryRef>
         <DirectoryRef Id="CustomPluginsBinFolder">
-            <Component Id="CMP_V1_CUSTOM_PLUGINS_BIN" Guid="6a493ba5-6d01-4b1d-9d1c-27c231fc3d4c" KeyPath="yes">
+            <Component Id="CMP_V1_CUSTOM_PLUGINS_BIN" Guid="e4509a7a-409e-4a38-8705-37de9cf0d72d" KeyPath="yes">
                 <CreateFolder/>
             </Component>
         </DirectoryRef>
     </Fragment>
 
     <Fragment>
         <ComponentGroup Id="CustomPluginsBinComponent" Directory="CustomPluginsBinFolder">
-            <Component Id="CMP_NRI_$(var.IntegrationName)_EXE" Guid="9f89c600-fd0e-406d-8e79-8d1e402a4393" Win64="no">
+            <Component Id="CMP_NRI_$(var.IntegrationName)_EXE" Guid="5cd4cba9-75b1-41e8-8bc8-36b267850f57" Win64="no">
                 <File Id="FILE_NRI_$(var.IntegrationName)_EXE"
                       Source="$(var.BinariesPath)New Relic\newrelic-infra\newrelic-integrations\bin\nri-$(var.IntegrationName).exe"
                       KeyPath="yes"/>
             </Component>
-            <Component Id="CMP_NRI_$(var.IntegrationName)_LICENSE" Guid="4d336ed6-54be-46d9-ad9f-700259a31201" Win64="no">
+            <Component Id="CMP_NRI_$(var.IntegrationName)_LICENSE" Guid="806ceabd-ffeb-4d6a-a8e4-298dd3474f54" Win64="no">
                 <File Id="FILE_NRI_$(var.IntegrationName)_LICENSE"
                       Source="$(var.ProjectRootPath)LICENSE"
                       KeyPath="yes"/>
             </Component>
         </ComponentGroup>
-        <!-- <ComponentGroup Id="CustomPluginsComponent" Directory="CustomPluginsFolder">
-            <Component Id="CMP_NRI_$(var.IntegrationName)_DEFINITION_YML" Guid="f6961b3c-5fe9-45c8-bfe6-2fe78322f921" Win64="no">
-                <File Id="FILE_NRI_$(var.IntegrationName)_DEFINITION_YML"
-                      Name="$(var.IntegrationName)-win-definition.yml"
-                      Source="$(var.BinariesPath)New Relic\newrelic-infra\newrelic-integrations\$(var.IntegrationName)-win-definition.yml"
-                      KeyPath="yes"/>
-            </Component>
-        </ComponentGroup>
-        <ComponentGroup Id="PluginConfigsComponent" Directory="PluginConfigsFolder">
-            <Component Id="CMP_NRI_$(var.IntegrationName)_CONFIG_YML" Guid="829cd54e-8454-44c6-aec6-9e6b38373717" Win64="no"
-                       NeverOverwrite="yes" Permanent="yes">
-                <File Id="FILE_NRI_$(var.IntegrationName)_CONFIG_YML"
-                      Name="$(var.IntegrationName)-win-config.yml.sample"
-                      Source="$(var.BinariesPath)New Relic\newrelic-infra\integrations.d\$(var.IntegrationName)-win-config.yml.sample"
```

Comment:
looks this is not the [config](https://github.com/newrelic/nri-prometheus/blob/main/configs/nri-prometheus-config.yml.sample) in the repo . Why we are not adding configs to msi?

Reply:
That was only a comment that I removed. 

I believe that we are not adding it since it is going to be generated and the user should not configure it.
Currently Linux behaves in the same way (no config files pre-packaged), therefore I believe it was a decision and not a mistake.

@arvdias @ardias Do you remember why we were not packaging configs?

Reply:
yes you are right , this is bundle with the agent. We can include it later if needed